data/json/items/tools.json

```diff
@@ -7949,6 +7979,16 @@
     "to_hit": -2,
     "flags": ["RADIO_MODABLE", "RADIO_INVOKE_PROC", "BOMB"],
     "category": "weapons",
+    "explode_in_fire" : true,
+    "explosion" : {
+        "power": 60,
+        "shrapnel": {
+            "count": 30,
+            "mass": 10,
+            "recovery": 20,
+            "drop": "scrap"
```

Comment:
The description mentions that it is indeed a plastic jug, but one that is, quote, "filled with black gunpowder and scrap metal". Hence the addition of scrap shrapnel, and scrap metal to the recipe.